src/views/dashboard/logs/query/SQLBuilder.vue

```diff
@@ -0,0 +1,159 @@
+<template lang="pug">
+a-form(
+  label-align="left"
+  size="small"
+  auto-label-width
+  :model="form"
+)
+  a-form-item(field="table" label="Table")
+    a-select(
+      v-model="inputTableName"
+      style="width: auto"
+      placeholder="Select Table"
+      :options="tables"
+      :allow-search="true"
+      @change="handleTableChange"
```

Comment:
```suggestion
       :trigger-props="{ autoFitPopupMinWidth: true }"
       @change="handleTableChange"
```
Recommend adding this props. It will allow tables in options list to show the full name.